data.json

```
@@ -2095,6 +2095,16 @@
                 "Ansible"
             ],
             "description": "Exosphere is a user-friendly client interface for OpenStack-based cloud systems."
+        },
+        {
+            "name": "SuperDuperDB",
+            "link": "https://github.com/SuperDuperDB/superduperdb",
+            "label": "Good First Issue",
```

```suggestion
            "label": "good first issue",
```

Otherwise looks all good